homeassistant/components/sensor/tautulli.py

```diff
@@ -119,7 +119,10 @@ def name(self):
     @property
     def state(self):
         """Return the state of the sensor."""
-        return self.sessions['stream_count']
+        try:
+            return self.sessions['stream_count']
+        except KeyError:
+            return -1
```

Comment:
How about just using `dict.get(key)`?
```
return self.sessions.get('stream_count')
```
This will result in `None` or "Unkown" in the UI.
![image](https://user-images.githubusercontent.com/15093472/51003924-e1abdf00-1538-11e9-87d6-62ceade6e516.png)